### DISCUSSION TITLE
请问插件“设置”-“内容”页面中，“忽略段落”、“忽略元素”、“保留元素”三个设置的区别？

### DISCUSSION THREAD

**Discussion by GoTop** [bookfere/Ebook-Translator-Calibre-Plugin#280] [Q&A]
请问插件“设置”-“内容”页面中，“忽略段落”、“忽略元素”、“保留元素”三个设置的区别？

**Comment by bookfere** [comment]
刚刚在设置界面上添加了提示：

**忽略段落**：不要翻译包含这些关键词的元素。
**忽略元素**：不要翻译匹配这些规则的元素。
**保留元素**：保留已抽取元素中匹配这些规则的元素。

**Reply by GoTop** [reply]
就是这个忽略和保留在翻译书籍时，具体区别是什么呢？能举个例子吗？

**Reply by bookfere** [reply]
```html
<div>
    <p class="test">b</p>
    <p>a</p>
    <p>c <span class="test">d<span> e</p>
</div>
```

如果在“忽略元素”中添加了 `.test`，第一个 `p` 元素就会被忽略。

如果在“忽略段落”中添加了 `a` 这条规则，第二个 `p` 元素就会被忽略。

如果你在“保留元素”中添加了 `span.test`，`span` 元素将会被原样保留不翻译。

**Reply by GoTop** [reply]
被忽略的效果是在输出成双语书的时候，原文中有这个元素——b和a，但是译文里直接不显示这个元素？

而保留元素，效果是在输出双语书的时候，在原文、译文中都会有这个个元素，而且在译文中，这个元素是不会被翻译的
